tools/cldr-apps/js/src/esm/cldrNotify.mjs

```diff
@@ -101,12 +131,22 @@ function errorWithCallback(message, description, callback) {
 }
 
 /**
- * Display an error notification for an exception
+ * Log and Display an error notification for an exception
  *
  * @param {Object|String} e the Error that was thrown and caught
  * @param {String} context a description of where it was caught
  */
 function exception(e, context) {
+  e = logException(e, context);
+  notification.error({
+    message: "Internal error: " + e.name + " " + context,
+    description: e.message,
+    duration: NO_TIMEOUT,
+  });
```

Comment:
Just a thought for the future: we might in general have two error messages, one for ordinary users and one including "technical details", and the latter might be available by clicking a button for "technical details"